patch/cavm-as7512/config-accton-as7512_32x.patch

```diff
@@ -0,0 +1,71 @@
+
+accton driver config, i2c_mux_pca954x and pmbus.
+From: roy_lee <roy_lee@accton.com>
+
+---
+ debian/build/build_amd64_none_amd64/.config | 23 ++++++++++++++++++++---
+ 1 file changed, 20 insertions(+), 3 deletions(-)
+
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index c0b11a1..df010e7 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -1788,6 +1788,8 @@ CONFIG_EEPROM_LEGACY=m
+ CONFIG_EEPROM_MAX6875=m
+ CONFIG_EEPROM_93CX6=m
+ # CONFIG_EEPROM_93XX46 is not set
++CONFIG_EEPROM_ACCTON_AS7512_32x_SFP=m
++CONFIG_EEPROM_SFF_8436=m
```

Comment:
this is already in https://github.com/Azure/sonic-linux-kernel/blob/master/patch/config-dell-s6000.patch, please check